### DISCUSSION TITLE
Helpful APIs : Add the APIs you find

### DISCUSSION THREAD

**Discussion by PrajjwalDatir** [PrajjwalDatir/Kaoi#206] [Show and tell]
Add the APIs, you find online.

**Comment by PrajjwalDatir** [comment]
https://github.com/simsimi/SmallTalk-API

**Comment by PrajjwalDatir** [comment]
https://api.xteam.xyz/

**Comment by Raj6544** [comment]
https://rapidapi.com/brianiswu/api/genius

**Comment by Raj6544** [comment]
https://rapidapi.com/stujo/api/tic-tac-toe

**Comment by Raj6544** [comment]
https://rapidapi.com/orthosie/api/riddles

**Comment by Raj6544** [comment]
https://rapidapi.com/gaurmanojkumar530/api/text-analysis12

**Comment by Raj6544** [comment]
https://rapidapi.com/hmerritt/api/imdb-internet-movie-database-unofficial

**Comment by Raj6544** [comment]
https://some-random-api.ml/endpoints
